packages/spindle-ui/src/Form/Checkbox.css

```diff
@@ -57,15 +57,10 @@
 }
 
 /* box-shadow is used to apply border-radius as outline */
-.spui-Checkbox-input:focus ~ .spui-Checkbox-outline,
-.spui-Checkbox-input:focus-visible ~ .spui-Checkbox-outline {
+.spui-Checkbox-input:focus ~ .spui-Checkbox-outline {
   box-shadow: 0 0 0 2px var(--color-focus-clarity);
 }
 
-.spui-Checkbox-input:focus:not(:focus-visible) ~ .spui-Checkbox-outline {
```

Comment:
@masuP9 I'm not sure why this rule was deleted. Is this better for the selectable elements?

Reply:
@herablog Oh, I mistaken, it looked to me at local env that checkbox and radio always apply focus-visible. I will fix.